SLAM/iterative_closest_point/iterative_closest_point.py

```diff
@@ -152,6 +148,29 @@ def main():
         print("R:", R)
         print("T:", T)
 
+    # test for 3d point set
```

Comment:
I think this test for 3d point test should be separated to main(), like main_3d_point() or something. And then iterative_closest_point.py should be updated.